dev-perl/Statistics-Basic/Statistics-Basic-1.661.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DIST_AUTHOR=JETTERO
+DIST_VERSION=1.6611
+inherit perl-module
+
+DESCRIPTION="Statistics-Basic module"
+
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~sparc ~x86"
+IUSE=""
+
+DEPEND="dev-perl/Number-Format"
+RDEPEND="${DEPEND}"
+
+SRC_TEST=do
```

Comment:
This line been invalid (and redundant) since EAPI6.

Tests are on by default now ;)